src/azure/storage/blob/block_with_size_list.rs

```diff
@@ -46,10 +45,8 @@ where
     pub blocks: Vec<BlobBlockWithSize<T>>,
 }
 
-impl<'a> TryFrom<&'a str> for BlockWithSizeList<String> {
```

Comment:
This is the only change, instead of implementing the nightly-only trait I've migrated the function to struct implementation